src/plugins/auto/auto_schedule.cpp

```diff
@@ -459,15 +567,21 @@ bool AutoSchedule::ScheduleToWorkerInferRequest(IE::Task inferPipelineTask, Devi
         }
         devices.push_back(_loadContext[ACTUALDEVICE].deviceInfo);
     } else {
-        // _acceleratorDevice could be the same as _cpuDevice, such as AUTO:CPU
-        if (_loadContext[ACTUALDEVICE].isAlready) {
-            devices.push_back(_loadContext[ACTUALDEVICE].deviceInfo);
+        if (_pCTPUTLoadContext) {
+            for (size_t i = 0; i < _autoSContext->_devicePriorities.size(); i++) {
+                devices.push_back(_autoSContext->_devicePriorities[i]);
```

Comment:
@wangleis do we need to support runtime change of device priority list with auto ctput?

Reply:
if yes, we need to handle this line thread-safe.

Reply:
@songbell  Runtime priority change is planned with round robin policy in following PR after 2023.0.

Reply:
see, if then, I suppose related docs which involves this functionality needs to be updated?